admin/address-report.vbs

```diff
@@ -0,0 +1,259 @@
+objExcel.Cells(1, 10).Value = "MAILING ADDRESS LINE 2"
+objExcel.Cells(1, 11).Value = "MAILING CITY"
+objExcel.Cells(1, 12).Value = "MAILING STATE"
+objExcel.Cells(1, 13).Value = "MAILING ZIP CODE"
+objExcel.Cells(1, 14).Value = "HOMELESS"
+
+FOR i = 1 to 14		'formatting the cells'
+	objExcel.Cells(1, i).Font.Bold = True		'bold font'
+	ObjExcel.columns(i).NumberFormat = "@" 		'formatting as text
+	objExcel.Columns(i).AutoFit()				'sizing the columns'
+NEXT
+
+excel_row = 2
+back_to_self	'Does this to prevent "ghosting" where the old info shows up on the new screen for some reason
+Call navigate_to_MAXIS_screen("REPT", "ACTV")
+
+For each worker in worker_array
+    Call write_value_and_transmit(worker, 21, 13)
+	EMReadScreen has_content_check, 1, 7, 8     	'Skips workers with no info
+	If has_content_check <> " " then
+		Do                          
+			MAXIS_row = 7  			'Set variable for next do...loop
+			EMReadScreen last_page_check, 21, 24, 2		'Checking for the last page of cases because on REPT/ACTV it displays right away, instead of when the second F8 is sent
+			Do
+				EMReadScreen MAXIS_case_number, 8, MAXIS_row, 12		'Reading case number
+				MAXIS_case_number = trim(MAXIS_case_number)
+                If MAXIS_case_number = "" then exit do			'Exits do if we reach the end
+                capture_case = False 
+                EMReadScreen cash_program, 1, MAXIS_row, 54
+                EmReadScreen cash_program_2, 1, MAXIS_row, 59
+                EMReadScreen SNAP_program, 1, MAXIS_row, 61
+                
+                If cash_program = "A" then capture_case = True 
+                If cash_program_2 = "A" then capture_case = True 
+                If SNAP_program = "A" then capture_case = True 
+
+                If capture_case = True then 
+                    EMReadScreen client_name, 21, MAXIS_row, 21		'Reading client name    
+				    ObjExcel.Cells(excel_row, 1).Value = worker
+				    ObjExcel.Cells(excel_row, 2).Value = MAXIS_case_number
+				    ObjExcel.Cells(excel_row, 3).Value = trim(client_name)
+				    excel_row = excel_row + 1
+                End if 
+				MAXIS_row = MAXIS_row + 1
+				MAXIS_case_number = ""			'Blanking out variable
+                STATS_counter = STATS_counter + 1
+			Loop until MAXIS_row = 19
+			PF8
+		Loop until last_page_check = "THIS IS THE LAST PAGE"
+	End if
+Next
+
+'formatting excel columns to fit
+FOR i = 1 to 14
+	objExcel.Columns(i).AutoFit()
+NEXT
+
+'Filling in STAT/ADDR information for SNAP and/or Cash programs that are in an active status. 
+excel_row = 2
+Do
+	MAXIS_case_number = ObjExcel.Cells(excel_row, 2)    'Assign case number from Excel
+	If MAXIS_case_number = "" then exit do  	'Exiting if the case number is blank
+	Call navigate_to_MAXIS_screen_review_PRIV("STAT", "ADDR", is_this_priv)  'Navigate to stat/addr to grab address - checking for PRIV
+    If is_this_priv = True then
+        objExcel.Cells(excel_row, 4).Value = "Privileged"
```

Comment:
Need an incrementor of the excel_row after this line or this would get stuck in a loop here.